lib/entities/context/DutchQuoteContext.ts

```diff
@@ -200,7 +200,7 @@ export class DutchQuoteContext implements QuoteContext {
 
     const syntheticStatus = await this.syntheticStatusProvider.getStatus(this.request.info);
     const chainId = classicQuote.request.info.tokenInChainId;
-    const quoteConfig = ChainConfigManager.getQuoteConfig(chainId, this.request.routingType);
+    const quoteConfig = ChainConfigManager.getQuoteConfig(chainId, this.request.routingType) as DutchConfig;
```

Comment:
Why do we have to cast to DutchConfig here? I would imagine that passing in the routingType into this func should be enough to return a fully typed object

Reply:
Yeah casting makes me think theres some improvements we can do to the typing system